src/openvic-simulation/utility/StringUtils.hpp

```diff
@@ -1,10 +1,11 @@
 #pragma once
 
+#include <algorithm>
```

Comment:
Why change includes?
Did you mean to include code changes or was there a compilation issue or something with intellisense?

Reply:
This was added to make intellisense calm down about something, also in general it's good practice to include stuff directly in the file even if it's also included via other headers. I'm not as strict about it as I should be, Spartan's usually the most on it.